Qt 5.11 cleanes up the internal headers and so consumers could fail by
missing includes
--- a/src/gui/entry/EditEntryWidget.cpp
+++ b/src/gui/entry/EditEntryWidget.cpp
@@ -32,6 +32,7 @@
 #include <QMimeData>
 #include <QEvent>
 #include <QColorDialog>
+#include <QButtonGroup>
 
 #include "autotype/AutoType.h"
 #include "core/Config.h"
